reviews/review-for-contributors.md

```diff
@@ -0,0 +1,45 @@
+* Please avoid force-pushing once review has started, except in the circumstances listed below; this tends to disrupt incremental review and is not supported well enough by GitHub's reviewing tools. In particular, never force-push several times in a row; if you did, GitHub would to merge the events in the PR's timeline, showing only details of the last force-push, making it impossible to find again which commit was last reviewed. Exceptions:
+  * As a general rule, before you force-push, check with the reviewers if it is OK with them, and make a note in a GitHub comment with the hash of the commit that was last reviewed.
+  * The most common reason for force-pushing is when conflicts with the base branch have appeared. We usually resolve these by rebasing. (Though, for complex PRs, merging the base branch into the PR can be considered; this should be discussed with reviewers too.)
+  * It is generally acceptable to force-push simple fixes (such as typos) to the last commit (but only that one) in the PR shortly after it was pushed.
+  * Once a PR is approved, some people like to have a zero-diff force-push to clean up the history (like, fixing typos or syntax errors in the commit they were introduced, etc.). This is not a general rule and should be discussed with the reviewers. We like to keep a history that reflects the evolution of the code; in particular we don't generally squash the entire PR to a single commit before merging.
+* If you have appropriate permissions on GitHub, when you think you have addressed feedback and the PR is ready for the next round of reviews, please update labels (remove "needs-work" and add "needs-review") and re-request review from existing reviewers. Otherwise, please leave a comment.
+
+## Creating backports
+
+Some changes need to be backported to the Long-Term Support (LTS) branches: security fixes, bug fixes, and some testing improvements. In that case, as a general rule we wait until the backport is ready and approved before merging the main PR.
+
+A backport is a separate PR that, instead of targeting `development` as usual, targets a LTS branch. That secondary PR should have the same title as the main one with "[Backport x.yy] " prepended, where x.yy is the name of LTS branch (for example 2.28). The description should start with "This is the x.yy backport of #zzzz", where zzzz is the number of the original PR, and then mention if the backport is trivial, or if there were conflicts that had to be resolved, or important differences with the main branch (for example: "only contextual conflicts", or "had to add/remove XXX because feature YYY is present only on one side", or "skipped commits U and V because they were irrelevant").
```

Comment:
> with "[Backport x.yy] " prepended

Minor: we use "Backport x.yy:" as the prefix about three times as often (stats from 2.28).

Reply:
Ah, didn't notice, I always use []. Thanks for checking!